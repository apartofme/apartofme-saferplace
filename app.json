{
  "name": "SaferPlace",
  "displayName": "Safer Place"
}
