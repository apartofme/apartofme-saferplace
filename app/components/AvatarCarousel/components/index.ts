export * from './AvatarCarouselItem';
export * from './ProgressBarItem';
