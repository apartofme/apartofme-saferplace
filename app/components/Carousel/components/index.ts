export * from './SubtitleImage/SubtitleImage';
export * from './ImageTitleSubTitle';
export * from './ImageSubtitle';
export * from './OnlyImage';
export * from './ProgressBarItem';
