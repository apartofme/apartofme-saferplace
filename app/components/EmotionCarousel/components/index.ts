export * from './EmotionCarouselItem';
export * from './ProgressBarItem';
