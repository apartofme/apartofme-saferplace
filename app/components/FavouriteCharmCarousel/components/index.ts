export * from './FavouriteCharmCarouselItem';
export * from './ProgressBarItem';
