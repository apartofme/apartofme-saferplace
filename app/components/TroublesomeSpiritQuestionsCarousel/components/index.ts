export * from './TroublesomeSpiritQuestionsCarouselItem';
export * from './ProgressBarItem';
