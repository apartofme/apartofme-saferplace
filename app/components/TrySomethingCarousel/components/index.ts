export * from './TrySomethingCarouselItem';
export * from './ProgressBarItem';
