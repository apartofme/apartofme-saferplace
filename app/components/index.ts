export * from './ExtendedButton';
export * from './ExtendedText';
export * from './ExtendedTextInput';
