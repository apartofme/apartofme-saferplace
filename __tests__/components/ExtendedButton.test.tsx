import 'react-native';
import React from 'react';
import testRenderer from 'react-test-renderer';

import { ExtendedButton } from '../../app/components';

test('UI: ExtendedButton should match snapshot', () => {
  const actualExtendedButton = testRenderer
    .create(<ExtendedButton title="hello, test" />)
    .toJSON();
  expect(actualExtendedButton).toMatchSnapshot();
});
