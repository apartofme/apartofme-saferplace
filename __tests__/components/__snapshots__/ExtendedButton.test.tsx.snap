// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI: ExtendedButton should match snapshot 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  nativeID="animatedComponent"
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#333",
      "opacity": 1,
      "padding": 16,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#333",
          "fontSize": 15,
        },
        Array [
          Object {
            "color": "white",
          },
          undefined,
        ],
      ]
    }
  >
    hello, test
  </Text>
</View>
`;
